ext/RastersGRIBDatasetsExt/gribdatasets_source.jl

```diff
@@ -14,3 +12,7 @@ function RA._open(f, ::Type{GRIBsource}, filename::AbstractString; write=false,
     ds = GRIBDatasets.GRIBDataset(filename)
     RA._open(f, GRIBsource, ds; kw...)
 end
+
```

Comment:
@tcarion @Alexander-Barth this is the hack that is currently required to make disk arrays chunking propagate out from the internal `Varable` or `DataValues` objects.

Note that with your current implementation this means anyone using `CFVariable` can not do chunked reads of large datasets. The whole thing is read at once.